packages/api-server/package.json

```diff
@@ -8,37 +8,56 @@
     "directory": "packages/api-server"
   },
   "license": "MIT",
-  "main": "dist/cliHandlers",
+  "exports": {
+    ".": {
+      "types": "./dist/cliConfig.d.ts",
+      "default": "./dist/cliConfig.js"
+    },
+    "./create-server": {
+      "types": "./dist/createServer.d.ts",
+      "default": "./dist/createServer.js"
+    },
+    "./fastify-api": {
+      "types": "./dist/plugins/api.d.ts",
+      "default": "./dist/plugins/api.js"
+    },
+    "./fastify-graphql": {
+      "types": "./dist/plugins/graphql.d.ts",
+      "default": "./dist/plugins/graphql.js"
+    },
+    "./helpers": {
+      "types": "./dist/helpers.d.ts",
+      "default": "./dist/helpers"
+    },
```

Comment:
I have an idea I'd like to float by you @Tobbe instead of going with this pattern again; I did this so that I could share `getPort/getHost` functionality with the CLI like we did for `coerceRootPath`